torch/csrc/jit/script/compiler.cpp

```diff
@@ -259,6 +325,7 @@ struct Environment {
         {"bool", std::make_shared<CastValue>(BoolType::get(), prim::Bool)},
         {"getattr", std::make_shared<GetAttrValue>()},
         {"isinstance", std::make_shared<IsInstanceValue>()},
+        {"_unchecked_unwrap_optional", std::make_shared<UncheckedUnwrapOptional>()},
```

Comment:
There needs to be special casing of the duplicate unwrapped optional calls, and this is the most self-contained way to achieve it. Additionally, we can't just not print out the unchecked_optionals because then we lose the boolean expressions, since those get desugared to if statements after export/import.
e.g.:
if x is None and len(x) < 2:
   print("hi")

On the second compilation, the compiler won't know to combine " x is None and len(x)" since they will no longer be in a boolean expression. But the inserted unwrap optional from the first compilation will still exist.